lib/TestBootstrap.lua

```diff
@@ -69,14 +69,18 @@ end
 	the test plan before we execute it, allowing them to toggle specific tests
 	before they're run, but after they've been identified!
 ]]
-function TestBootstrap:run(root, reporter, showTimingInfo, noXpcallByDefault)
-	noXpcallByDefault = noXpcallByDefault or false
+function TestBootstrap:run(root, reporter, otherOptions)
+	reporter = reporter or TextReporter
+
+	otherOptions = otherOptions or {}
+	local showTimingInfo = otherOptions["showTimingInfo"]
+	local noXpcallByDefault = otherOptions["noXpcallByDefault"] or false
```

Comment:
We should make `showTimingInfo` fall back in the same way `noXpcallByDefault` does.